rllib/policy/torch_policy.py

```diff
@@ -317,9 +317,9 @@ def compute_actions_from_input_dict(
             # Calculate RNN sequence lengths.
             seq_lens = (
                 torch.tensor(
-                    [1] * len(input_dict["obs"]),
+                    [1] * len(state_batches[0]),
```

Comment:
is this also a bug in the tf dynamic and eager policies? Could you inspect that and make a similar change for those if so? thanks!

Reply:
I verified the code for eager_tf_policy.py and it is handling the seq_lens parameter correctly.
You can verify it here: https://github.com/ray-project/ray/blob/7d262f886dec5c68fe3270aea1492aa483048a68/rllib/policy/eager_tf_policy.py#L758

Thanks :D